styles/variables-silicon.less

```diff
@@ -185,39 +185,36 @@
 
 // Input
 // ---------------------------------------
-@agate-input-border-width: 3px;
+@agate-input-decorator-border-bottom-width: 3px;
 @agate-input-decorator-border-width: 0;
+@agate-input-decorator-disabled-opacity: 0.3;
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/115637068-5130eb80-a34a-11eb-9721-e8e37dc0c2b7.png)

 Is this correct opacity for disable?

Reply:
Sorry I missed this, I fixed it.